Packs/HealthCheck/Scripts/HealthCheckPlaybookAnalysis/HealthCheckPlaybookAnalysis.py

```diff
@@ -89,42 +89,68 @@ def findTopUsedPLaybooks(accountName):
 builtinPlaybooksNames = []
 res = []
 if customPlaybooks is not None:
+    copyDetected = []
+    sleepDetected = []
+    multiSetIncidentDetected = []
+    multiTasksDetected = []
+    emailAskUserDetected = []
+
     for builtinPlaybook in builtinPlaybooks:
         builtinPlaybooksNames.append(builtinPlaybook["name"])
 
     for customPlaybook in customPlaybooks:
 
         for builtinPlaybooksName in builtinPlaybooksNames:
             if builtinPlaybooksName in customPlaybook["name"]:
-                res.append({"category": "Playbooks", "severity": "Low",
-                            "description": f"{DESCRIPTION[0]}".format(customPlaybook["name"]),
-                            "resolution": f"{RESOLUTION[0]}"
-                            })
+                copyDetected.append(customPlaybook["name"])
 
         if "Sleep" in customPlaybook["scriptIds"]:
-            res.append({"category": "Playbooks", "severity": "Low",
-                        "description": f"{DESCRIPTION[1]}".format(customPlaybook["name"]),
-                        "resolution": f"{RESOLUTION[1]}"
-                        })
+            sleepDetected.append(customPlaybook["name"])
 
         if str(customPlaybook).count("Builtin|||setIncident") >= thresholds['CustomPlaybookSetIncidentCount']:
-            res.append({"category": "Playbooks", "severity": "Low",
-                        "description": f"{DESCRIPTION[2]}".format(customPlaybook["name"]),
-                        "resolution": f"{RESOLUTION[2]}"
-                        })
+            multiSetIncidentDetected.append(customPlaybook["name"])
 
         if "EmailAskUser" in customPlaybook["scriptIds"]:
-            res.append({"category": "Playbooks", "severity": "Low",
-                        "description": f"{DESCRIPTION[3]}".format(customPlaybook["name"]),
-                        "resolution": f"{RESOLUTION[3]}"
-                        })
+            emailAskUserDetected.append(customPlaybook["name"])
 
         if len(customPlaybook["tasks"]) > thresholds['CustomPlaybookLength']:
-            res.append({"category": "Playbooks", "severity": "Low",
-                        "description": f"{DESCRIPTION[4]}".format(customPlaybook["name"]),
-                        "resolution": f"{RESOLUTION[4]}"
-                        })
+            multiTasksDetected.append(customPlaybook["name"])
+
+
+res = []
+if copyDetected:
+    res.append({"category": "Playbooks", "severity": "Low",
+                "description": f"{DESCRIPTION[0]}".format(", ".join(copyDetected)),
+                "resolution": f"{RESOLUTION[0]}"
```

Comment:
Relevant for all the following blocks as well.
```suggestion
                "description": DESCRIPTION[0].format(", ".join(copyDetected)),
                "resolution": RESOLUTION[0]
```

Reply:
fixed as suggested